src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommandProviders.cs

```diff
@@ -62,7 +62,9 @@ public class RestoreCommandProviders
             ILogger log)
         {
             var isFallbackFolder = false;
-            var globalPackages = new NuGetv3LocalRepository(globalFolderPath, packageFileCache, isFallbackFolder);
+            var updateLastAccess = true; // make configurable
```

Comment:
Is this meant to be a TODO?

Reply:
Kinda. I wasn't sure whether I was supposed to implement something here or not, given that this `Create` method in `RestoreCommandProviders.cs` is only referenced in tests and `GlobalSuppressions.cs`. So I guess I will remove this change.